plugins/modules/dcnm_policy.py

```diff
@@ -1251,7 +1437,7 @@ def dcnm_translate_config(self, config):
                     updated_config.append(cfg)
             config = updated_config
         else:
-            config = override_config
+            config += override_config
```

Comment:
Is this working? When I checked it said '+=' not supported for dicts. Suggested to use update instead.

Reply:
`config` and `override_config` are lists, not dicts, and yes, it works